### ISSUE TITLE
Python 3.11

### ISSUE THREAD

**Issue by ritapf-sa** [SBentley/qvd-utils#18]
Hello!
Please compile the library for python 3.11 if it's possible, thank you so much for making this tool

**Comment by ShepTeck** [comment]
Yes, Please!  I am no longer allowed to install Python 3.7 or 3.9 at my job and really need this!
